go.mod

```diff
@@ -0,0 +1,10 @@
+module health-check
```

Comment:
Suggest follow project convention of `github.com/edgexfoundry/docker-edgex-consul/health` or something like that.